locales/README.md

```diff
@@ -0,0 +1,16 @@
+# Locales
+
+## Note to contributors
+
+Please change only the locale content in the `en-US`
+directory. The other directories are changed using [Pontoon](https://pontoon.mozilla.org/projects/firefox-profiler/).
+
+Here are some simple rules:
+1. If you change the string in a way that changes the meaning, you need to
+   change the string key as well. If that doesn't change the meaning (for
+   example, if that's a typo fix), then the string key will stay the same.
```

Comment:
```suggestion
   example, if that's a typo fix), then the string key can stay the same.
```

Somewhere in this paragraph, I would add a link to https://mozilla-l10n.github.io/documentation/localization/making_string_changes.html

Reply:
I added more links to Mozilla's l10n documentation, thanks for the reminder!